assets/css/base/gutenberg-blocks.scss

```diff
@@ -305,7 +305,12 @@
 	}
 
 	// Image
-	.wp-block-image {
+	.wp-block-image:not( .block-editor-media-placeholder ) {
+		
+		&:not( .block-editor-media-placeholder ) {
```

Comment:
why do we have double classes here? Wouldn’t one be enough?

Reply:
Agreed! I made this change and managed to push it to @nielslange branch (my plan was to set up a parallel branch in this repo so we can merge sooner).

Reply:
@senadir That was my bad. I manually added the change and then merged your suggestion as well, which lead to the duplicate code. Sorry for the confusion.